openapi.yaml

```diff
@@ -1,279 +1,678 @@
+          $ref: '#/components/schemas/ServiceLinks'
+    LocationsResourceObject:
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+        attributes:
+          $ref: '#/components/schemas/Attributes'
+        links:
+          $ref: '#/components/schemas/SelfLink'
+        relationships:
+          $ref: '#/components/schemas/RelationshipsInLocations'
+    ServicesResultObjects:
+      properties:
+        links:
+          $ref: '#/components/schemas/Links'
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/ServicesResourceObject'
+    ServicesResourceObject:
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+        attributes:
+          $ref: '#/components/schemas/ServicesAttributes'
+        links:
+          $ref: '#/components/schemas/ServiceLinks'
+        relationships:
+          $ref: '#/components/schemas/RelationshipsInServices'
+    SelfLink:
+      properties:
+        self:
+          type: string
+          description: Url to this resource / location.
+    Attributes:
+      properties:
+        name:
+          type: string
+          description: Name of the location.
+        abbreviation:
+          type: string
+        latitude:
+          type: number
+          format: float
+        longitude:
+          type: number
+          format: float
+        geometry:
+          properties:
+            type:
+              type: string
+              enum:
+                - Polygon
+                - MultiPolygon
+            coordinates:
+              description: Array of ring arrays for each physical structure of a location
+              type: array
+              items:
+                description: Array of points that make up a ring.
+                type: array
+                items:
+                  type: array
+                  items:
+                    type: number
+                    format: float
+        summary:
+          type: string
+        description:
+          type: string
+        address:
+          type: string
+          example: 12345 NE STREET RD
+        city:
+          type: string
+          example: Corvallis
+        state:
+          type: string
+          example: OR
+        zip:
+          type: string
+          example: '97330'
+        telephone:
+          type: string
+          example: 541-111-1111
+        fax:
+          type: string
+          example: 541-111-1111
+        thumbnails:
+          type: array
+          items:
             type: string
-            description: Name of pet
-            example: Hedwig
-          owner:
+            format: url
+            example: https://example.oregonstate.edu/image.jpg
+        images:
+          type: array
+          items:
             type: string
-            description: Name of owner
-            example: Harry Potter
-          species:
+            format: url
```

Comment:
You should use `uri` instead of `url`. Also, I recommend rename this attribute to `imageUris` or `imageLinks` or something similar to make it clear.